meta_review/dump_to_db.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from meta_review.models import Participant, Comment, Reaction
+
+
+def dump_all(participants, comments, reactions):
+    '""Dump all meta-review related data into database.""'
+    dump_participants(participants)
+    dump_comments(comments)
+    dump_reactions(reactions)
+
+
+def dump_participants(participants):
+    '""Dump participants data into Django database.""'
+    logger = logging.getLogger(__name__)
+    logger.info('dump participants data into database')
+    participants_list = []
+    for key, participant in participants.items():
+        participants_list.append(participant)
+    try:
+        # there's no way to do bulk update, so we delete and then create
+        # bulk delete all participants
+        Participant.objects.all().delete()
+        # bulk create all participants
+        Participant.objects.bulk_create(participants_list)
+    except Exception as ex:
+        logger.error('Something went wrong saving participants: %s' % ex)
+
+
+def dump_comments(comments):
+    '""Dump comments data into Django database.""'
+    logger = logging.getLogger(__name__)
+    logger.info('dump review comments data into database')
+    comments_list = []
+    for key, comment in comments.items():
+        comments_list.append(comment)
```

Comment:
do we need to store comments which have not been meta-reviewed?
or have they already been excluded somewhere else?

It would be helpful to avoid storing them, to reduce spreading content which is of low value.

Think GDPR ... what happens if we're asked to remove all information about a person -- the git history would need to be sanitised.  Best to only copy data that we can justify keeping a copy of.

Reply:
Addressed around https://github.com/coala/community/pull/143/files#diff-745f0342d354672f4a930f0a8bc86590R60